client/my-sites/domains/transfer-domain/index.jsx

```diff
@@ -30,6 +30,7 @@ import TrademarkClaimsNotice from 'components/domains/trademark-claims-notice';
 export class TransferDomain extends Component {
 	static propTypes = {
 		initialQuery: PropTypes.string,
+		useStandardBack: PropTypes.string,
```

Comment:
This should be a boolean and the conversion from string to boolean should be done at the controller level.